versions.cfg

```diff
@@ -10,11 +10,11 @@ flake8 = 2.3.0
 coverage = 3.7.1
 
 # Robot
-robotframework = 2.8.4
```

Comment:
@jensens it seems the upgrade of selenium made the robot test unstable:

https://travis-ci.org/kitconcept/pas.plugins.memberpropertytogroup/jobs/120601664

I briefly looked into it but couldn't find the problem... :(